build.gradle

```diff
@@ -31,9 +31,9 @@
 buildscript {
   ext {
     opensearch_group = "org.opensearch"
-    opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
+    opensearch_version = System.getProperty("opensearch.version", "3.0.0-alpha1-SNAPSHOT")
```

Comment:
U need to add more changes as discussed: https://github.com/opensearch-project/custom-codecs/pull/216. Also move the 9.12 codecs to backward compatible directory. And create new classes for 10.1